apps/finance/app/src/transfer-types.js

```diff
@@ -1,21 +1,12 @@
-export const All = Symbol('All')
-export const Incoming = Symbol('Incoming')
-export const Outgoing = Symbol('Outgoing')
+export const All = 'ALL_TRANSFER'
```

Comment:
In this case, we may still want to use symbols (prefer them over strings when possible).

There's another comment below about whether we could use these in the script, but otherwise, let's revert this back to being a symbol.